### ISSUE TITLE
First Post & Last Post is too limiting to be useful

### ISSUE THREAD

**Issue by IVXCVI** [fredrikloch/hugoscroll#9]
Wouldn't it be more useful to use the javascript for the fixed nav on the header buttons under the title? The `first` and `last` post concept is quite limiting for a useful navigation menu.

**Comment by fredrikloch** [comment]
I agree that it might be useful for some use-cases to add more navigational options. However as this is mainly intended to be a simple landing page I don't really see the need for complex navigation. I have therefore tried to maintain the simplicity.

**Comment by IVXCVI** [comment]
Any pointers on how to implement the functionality myself? I'm going through `index.js` and I haven't figured out how fn-item links up to and scrolls to each post. I'll share whatever work I do to accomplish my goal.

**Comment by fredrikloch** [comment]
I would start to look at the function that handles the header links that's clickable when you scroll down.

`$('.post-title').each(function () {
	var t = $(this).text(),
	    index = $(this).parents('.post-holder').index();
	$fnav.append("<a class='fn-item' item_index='"+index+"'>"+t+"</a>")
	$(this).parents('article').attr('id',t.toLowerCase().split(' ').join('-'));
	$('.fn-item').click(function () {
		var i = $(this).attr('item_index'),
			s = $(".post[item_index='"+i+"']")

		$('html, body').animate({
				scrollTop: s.offset().top
			}, 400);

	})
})`

**Comment by IVXCVI** [comment]
Thanks Frederik, I'll keep trying to deconstruct how it works. Comparing with a compiled index.html file is also quite helpful. I'll post back once I have a satisfactory solution. Cheers!

On March 29, 2017 2:11:08 AM CDT, Fredrik <notifications@github.com> wrote:
>I would start to look at the function that handles the header links
>that's clickable when you scroll down.
>
>`$('.post-title').each(function () {
>	var t = $(this).text(),
>	    index = $(this).parents('.post-holder').index();
>	$fnav.append("<a class='fn-item' item_index='"+index+"'>"+t+"</a>")
>	$(this).parents('article').attr('id',t.toLowerCase().split('
>').join('-'));
>	$('.fn-item').click(function () {
>		var i = $(this).attr('item_index'),
>			s = $(".post[item_index='"+i+"']")
>
>		$('html, body').animate({
>				scrollTop: s.offset().top
>			}, 400);
>
>	})
>})`
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/fredrikloch/hugoscroll/issues/9#issuecomment-290003283

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
